src/components/projects-summary/projects-summary.controller.ts

```diff
@@ -191,7 +191,7 @@ export class ProjectsSummaryController implements angular.IController {
   };
 
   public handleGettingStartedClick() {
-    var cb: any = this.ctrl.startGettingStartedTour();
+    var cb: any = this.ctrl.startTour();
```

Comment:
should we be making sure startTour has been passed into the directive, and hiding the button if startTour is undefined / not a function.  Really a question of whether the guided tour is optional.